[debug] [2019-04-12T03:37:20.059Z] ----------------------------------------------------------------------
[debug] [2019-04-12T03:37:20.061Z] Command:       /usr/bin/node /usr/local/bin/firebase serve
[debug] [2019-04-12T03:37:20.062Z] CLI Version:   6.5.2
[debug] [2019-04-12T03:37:20.062Z] Platform:      linux
[debug] [2019-04-12T03:37:20.062Z] Node Version:  v10.15.2
[debug] [2019-04-12T03:37:20.062Z] Time:          Fri Apr 12 2019 00:37:20 GMT-0300 (Brasilia Standard Time)
[debug] [2019-04-12T03:37:20.062Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-04-12T03:37:20.070Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-04-12T03:37:20.070Z] > authorizing via signed-in user
[debug] [2019-04-12T03:37:20.070Z] [iam] checking project fico-ufmg for permissions ["firebase.projects.get"]
[debug] [2019-04-12T03:37:20.072Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/fico-ufmg:testIamPermissions  

[debug] [2019-04-12T03:37:21.245Z] <<< HTTP RESPONSE 200
[debug] [2019-04-12T03:37:21.247Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/fico-ufmg  
 
[debug] [2019-04-12T03:37:22.230Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/mnt/8C7C090A7C08F12E/Side-Projects/Research Website - Polymer 2'...
[info] 
[debug] [2019-04-12T03:37:23.260Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:753:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-04-12T03:37:47.165Z] ----------------------------------------------------------------------
[debug] [2019-04-12T03:37:47.167Z] Command:       /usr/bin/node /usr/local/bin/firebase serve
[debug] [2019-04-12T03:37:47.167Z] CLI Version:   6.5.2
[debug] [2019-04-12T03:37:47.167Z] Platform:      linux
[debug] [2019-04-12T03:37:47.167Z] Node Version:  v10.15.2
[debug] [2019-04-12T03:37:47.168Z] Time:          Fri Apr 12 2019 00:37:47 GMT-0300 (Brasilia Standard Time)
[debug] [2019-04-12T03:37:47.168Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-04-12T03:37:47.176Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-04-12T03:37:47.176Z] > authorizing via signed-in user
[debug] [2019-04-12T03:37:47.177Z] [iam] checking project fico-ufmg for permissions ["firebase.projects.get"]
[debug] [2019-04-12T03:37:47.179Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/fico-ufmg:testIamPermissions  

[debug] [2019-04-12T03:37:48.298Z] <<< HTTP RESPONSE 200
[debug] [2019-04-12T03:37:48.302Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/fico-ufmg  
 
[debug] [2019-04-12T03:37:49.238Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/mnt/8C7C090A7C08F12E/Side-Projects/Research Website - Polymer 2'...
[info] 
[debug] [2019-04-12T03:37:50.405Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:753:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
